src/messages/mod.rs

```diff
@@ -403,7 +413,9 @@ impl SignedRoutingMessage {
             SecurityMetadata::None | SecurityMetadata::Partial(_) => {
                 Err(RoutingError::FailedSignature)
             }
-            SecurityMetadata::Single(security_metadata) => security_metadata.verify(&self.content),
+            SecurityMetadata::Single(security_metadata) => security_metadata
```

Comment:
Why that change? how did singe security metadata change?

Reply:
Remnants from the previous version. Will revert.